packages/gatsby-link/src/index.tsx

```diff
@@ -88,8 +118,11 @@ class GatsbyLink extends React.Component {
     instance.disconnect()
   }
 
-  handleRef(ref) {
-    if (this.props.innerRef && this.props.innerRef.hasOwnProperty(`current`)) {
+  handleRef(ref: HTMLAnchorElement | null): void {
+    if (
+      this.props.innerRef &&
+      Object.prototype.hasOwnProperty.call(this.props.innerRef, `current`)
```

Comment:
why was this change necessary?

Reply:
https://eslint.org/docs/rules/no-prototype-builtins